tests/framework/CMakeLists.txt

```diff
@@ -29,44 +29,14 @@ set(CMAKE_BUILD_TYPE "Debug")
 
 set_output_dirs("${PROJECT_BINARY_DIR}/tests")
 
-if (UNIX)
-  set(arch_defs "UNIX")
-  if (APPLE)
-    set(arch_defs ${arch_defs} "MACOS")
-  else (APPLE)
-    set(arch_defs ${arch_defs} "LINUX")
-  endif (APPLE)
-else (UNIX)
-  set(arch_defs "WINDOWS")
-endif (UNIX)
-
 find_package(DrMemoryFramework PATHS ${framework_dir})
 
 # As we'll be calling configure_DynamoRIO_{client,standalone} from within
 # a function scope, we must set the global vars ahead of time:
 configure_DynamoRIO_global(OFF ON)
 
-# We save pdb+dll space by not using libc (xref DRi#714).
```

Comment:
Umbra pulls in libc, so we can't do this anymore.

Reply:
On Windows too?

Reply:
I think so, as of dddcfc6aefaa936fe930043be9723f372716aaef right?

Reply:
That one may have unnecessarily changed Windows too.  If I re-instate `set(DynamoRIO_USE_LIBC OFF)` today and build umbra it builds fine and there are no unusual imports.  Missing symbols at runtime are usually a UNIX thing for ELF and !LD_BIND_NOW: on Windows the linker complains at link time.

Reply:
Gotcha, should I file an issue and fix in a later CL? Or should I just fix it in this CL (i.e. revert effects of dddcfc6 only for Windows)